File: kotlin/3.kt
fun main() {
    val num1: Float? = readLine()?.toFloatOrNull() ?: return // type MUST be set to Float
    val num2: Float? = readLine()?.toFloatOrNull() ?: return
    val total = num1!!*num2!!; // !! confirm that the value is not null
    // print out the result
    println("Your first number is $num1, your second number is $num2, and the totla is $total")
}

File: js/3.js
//3
const prompt = require("prompt-sync")({ sigint: true }); // this is requried in node

let num1 = prompt();
let num2 = prompt();

let total = parseFloat(num1)*parseFloat(num2);

console.info(`Your first number is ${num1} and your second number is ${num2} and your total is ${total}`)